src/fontra/client/css/tooltip.css

```diff
@@ -21,10 +21,10 @@
 
 [data-tooltip] {
   position: relative;
-  cursor: default;
 }
 
 [data-tooltip]:hover::before {
+  text-transform: capitalize;
```

Comment:
I don't think I like this. I prefer the text to start with a cap, but not each word. For example, I prefer "Edit local axes" over "Edit Local Axes".

Reply:
I removed this line of code and used 'capitalizeFirstLetter' instead.